.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish Docker image
```

Comment:
Change to a more suitable name for this task

Reply:
@phchen5 changed, please review again